package/skelcd-control-MicroOS.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Dec 18 08:31:09 UTC 2024 - Stefan Schubert <schubi@suse.de>
+
+- Added systemd.show_status=1 to kernel params.
```

Comment:
we well need some ID for rationale of change like jira entry, bug number or even github issue.

Reply:
I have added. Thanks !